epytope/EpitopePrediction/__init__.py

```diff
@@ -9,10 +9,8 @@
 
 """
 
-from epytope.Core.Base import AEpitopePrediction
```

Comment:
Did you test this? I would think that it should fail...

Reply:
tutorial predictions worked. It was greyed-out by pycharm, so I removed it

Reply:
Okay added this import again